compose.yaml

```diff
@@ -26,22 +23,22 @@ services:
 
     volumes:
       - .:/spot_skills # Mount the repository (a ROS workspace) to the container
-      - spot_driver_ws:/spot_driver_ws # Mount the Spot driver's workspace into the container
+#       - spot_driver_ws:/spot_driver_ws # Mount the Spot driver's workspace into the container
```

Comment:
Another branch will be needed to resolve these Spot driver inconsistencies: The current working plan is to use the ROS 1 driver, rather than ROS 2, meaning that this service will need to be redefined/replaced.

Reply:
See issue #7 for further details.